eng/performance/helix.proj

```diff
@@ -18,10 +18,32 @@
     </HelixCorrelationPayload>
   </ItemGroup>
 
+  <PropertyGroup>
+    <PartitionCount>5</PartitionCount>
+  </PropertyGroup>
   <ItemGroup>
+    <Partition Include="Partition0" Index="0" />
+    <Partition Include="Partition1" Index="1" />
+    <Partition Include="Partition2" Index="2" />
+    <Partition Include="Partition3" Index="3" />
+    <Partition Include="Partition4" Index="4" />
+  </ItemGroup>
+
+  <!-- 
+    Partition the Microbenchamrks project, but nothing else
+    TODO: only for oficial builds 
```

Comment:
Maybe this TODO could be solved by adding the partitions Ids and count to the `benchmark_jobs.yml`?

By doing so we could define:
- defaults used my ML.NET and Roslyn that don't require partitioning
- different values for official and PR buils


Sth like:

```yml
// part of eng/performance/benchmark_jobs.yml 
  partitionCount: 1     # optional -- the number of partitions
  frameworks: [ '0' ]    # optional -- partition ids, indexed from 0

// part of  azure-pipelines.yml for PR runs
  partitionCount: 2
  frameworks: [ '0', '1' ]

// part of  azure-pipelines.yml for official runs
  partitionCount: 5
  frameworks: [ '0', '1', '2', '3', '4' ]
```

Reply:
I actually decided that I'd just partition PRs as well, since it does have the possibility of saving us time. I will remove the TODO.